kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -80,56 +81,120 @@ define([
         return tableBody.appendChild(newRow);
     }
 
-    // Convert the table to a datatable object to get functionality
-    function renderTable(container, rowCount) {
-        return $(container.querySelector('table')).dataTable({
-            searching: false,
-            pageLength: dataTablePageLength,
-            lengthChange: false,
-            columnDefs: [
-                {
-                    // action row should not be orderable
-                    targets: 2,
-                    orderable: false,
-                },
-                {
-                    // logs row should not be orderable
-                    targets: 3,
-                    orderable: false,
-                },
-            ],
-            fnDrawCallback: () => {
-                // Hide pagination controls if length is less than or equal to table length
-                if (rowCount <= dataTablePageLength) {
-                    $(container.querySelector('.dataTables_paginate')).hide();
-                }
-            },
-        });
-    }
-
     /**
      * create a job state list instance
      *
      * the config should be an object with a property 'jobManager', which executes
-     * the job actions available as part of the job status table, and 'model', the
-     * Props object from the parent cell with information about job execution state
+     * the job actions available as part of the job status table, and 'toggleTab',
+     * a function used to view the results of finished jobs.
      *
      * @param {object} config
      * @returns jobStateList instance
      */
     function factory(config) {
         const widgetsById = {},
+            logWidgets = {},
             bus = Runtime.make().bus(),
             listeners = {},
-            { jobManager, model } = config,
+            { jobManager, toggleTab } = config,
             developerMode = config.devMode || devMode.mode;
 
-        if (!model || !model.getItem('exec.jobs.byId')) {
+        if (!jobManager.model || !jobManager.model.getItem('exec.jobs.byId')) {
```

Comment:
use the model stored in the jobManager object to avoid having to pass a load of references around